client/X11/xf_cliprdr.c

```diff
@@ -1886,12 +1900,14 @@ xf_cliprdr_server_format_data_response(CliprdrClientContext* context,
 
 		if (strcmp(clipboard->data_format_name, "FileGroupDescriptorW") == 0)
 		{
+#ifdef WITH_FUSE
 			/* Build inode table for FILEDESCRIPTORW*/
 			if (xf_cliprdr_fuse_generate_list(clipboard, data, size) == FALSE)
 			{
 				/* just continue */
 				WLog_WARN(TAG, "fail to generate list for FILEDESCRIPTOR");
 			}
+#endif
```

Comment:
Is there a point in handling `FileGroupDescriptorW` at all if there is no FUSE support? Why not wrap this entire
```c
if (strcmp(clipboard->data_format_name, "FileGroupDescriptorW") == 0)
```
condition in an `#ifdef`?